css/notes.css

```diff
@@ -138,20 +138,27 @@
 }
 
 #app-content .utils {
-    padding: 0 20px 0 20px;
-    line-height: 30px;
     position: fixed;
-    top: 45px;
-    right: 20px;
+    top: 58px;
+    right: 0;
```

Comment:
Looks fine for me in both Chrome and Firefox:
![capture du 2015-07-09 14-21-06](https://cloud.githubusercontent.com/assets/925062/8600640/304630ee-2667-11e5-8fa6-596b255d72aa.png)

@Henni?